modules/combined/test/tests/adaptive_timestepping/adapt_tstep_function_change_restart1.i

```diff
@@ -48,6 +48,7 @@
     decomposition_method = EigenSolution
     add_variables  = true
     generate_output = 'vonmises_stress'
+    temperature = temp
```

Comment:
I thought  in your new code, `temperature` variable is not needed for `StressDivergenceKernels`, is that correct? Why do we need to add this in the master action?

Reply:
Temperature still needs to be specified. In the new code omitting this will result in a warning!